### ISSUE TITLE
BKCommonLib 1.12.1 support

### ISSUE THREAD

**Issue by Rocologo** [bergerhealer/BKCommonLib#36]
##### Spigot version: 1.12.1

<hr>

#### Problem or bug:
BKCommonLib fails after I have installed Spigot 1.12.1 which was released yesterday.

[08:39:10] [Server thread/INFO]: [BKCommonLib] Loading BKCommonLib v1.12-v4
[08:39:11] [Server thread/WARN]: A class member of net.minecraft.server.v1_12_R1.TileEntity was not found!
[08:39:11] [Server thread/WARN]: Failed to find method public void load:a((CommonTagCompound) NBTTagCompound nbttagcompound); - Alternatives:
[08:39:11] [Server thread/WARN]:   - public void a(World arg0);
[08:39:11] [Server thread/WARN]:   - public void a(EnumBlockRotation arg0);
[08:39:11] [Server thread/WARN]:   - public void a(EnumBlockMirror arg0);
[08:39:11] [Server thread/WARN]:   - public void a(CrashReportSystemDetails arg0);
[08:39:11] [Server thread/WARN]:   - public void load(NBTTagCompound arg0);
[08:39:11] [Server thread/WARN]:   - public void A();
[08:39:11] [Server thread/WARN]:   - public void z();
[08:39:11] [Server thread/WARN]:   - protected void b(World arg0);
[0

**Comment by SlimeDog** [comment]
Yep. Apparently there is an internal test that blocks it.

**Comment by bergerkiller** [comment]
Oh so that is what you meant with generate a warning @SlimeDog. I swear BKCommonLib should either fail to enable or at least print a warning when used on an invalid Minecraft version. I'll see to it that that happens.

**Comment by bergerkiller** [comment]
It did show an informative error message and then shuts itself down. The message was outdated though, so I've updated that. But the logic still works, it just wasnt put in this issue ticket. https://github.com/bergerhealer/BKCommonLib/commit/58b26d17a695140728c3247b410c0f0f7bdfd009

It does print a bunch of irrelevant missing field/method warnings though, which is expected because its an unsupported version. Ive suppressed those here. https://github.com/bergerhealer/BKCommonLib/commit/12d8019da4bee2767637c5b46d48b6d2d2b7ceb1

Closing the issue ticket since the latest dev-build has 1.12.1 support, which I will push to spigot today.
